docs/usage-as-end-user.md

```diff
@@ -125,6 +125,36 @@ If you don't want to configure anything for the `cloudControllerManager` simply
 
 ## WorkerConfig
 
+Multiple zones can be configured for a worker group of a GCP Shoot. The minimum number of machines in every worker group should be equal to the number of zones configured for that worker-group. 
+
+> The predicate is - A worker group with N zones configured should have minimum N machines.
+
+⚠️ This is important because, as of today, Cluster Autoscaler does not support scale-from-zero on GCP. 
+
+The following YAML is a snippet of a `Shoot` resource:
+
+```
+workers:
+  - name: worker-vezh0
+    machine:
+      type: n1-standard-2
+      image:
+        name: gardenlinux
+        version: 318.8.0
+    maximum: 6
+    minimum: 2 # the value should be equal to the number of zones
```

Comment:
```suggestion
    minimum: 2 # the value should be equal or greater than the number of zones
```

Reply:
Why was this labeled as resolved when it is still not?

Reply:
Ok, I messed the things, but this can be applied also to line 128
> Multiple zones can be configured for a worker group of a GCP Shoot. The minimum number of machines in every worker group should be equal to the number of zones configured for that worker-group.